policyengine_us/variables/gov/states/ut/tax/income/credits/taxpayer_credit/personal_exemption.yaml

```diff
@@ -0,0 +1,14 @@
+description: UT personal exemption for dependents.
+values:
+  2020-01-01: 1_750
+  2022-01-01: 1_802
+metadata:
+  unit: currency-USD
+  period: year
+  name: ut_personal_exemption
+  label: UT personal exemption
+  reference:
+    - title: Utah.gov, Utah Income Taxes, Utah Personal Exemption
```

Comment:
This shows the 2022 value of 1802 but would be good to have a tax form since this page changes each year